package-lock.json

```diff
@@ -0,0 +1,11339 @@
+{
```

Comment:
We gitignored the package-lock in master, can you sync with that so we can remove this?